Composer/packages/client/src/components/GetStarted/GetStartedNextSteps.tsx

```diff
@@ -0,0 +1,310 @@
+        label: formatMessage('Train your language model'),
+        description: formatMessage('Ensure your bot can understand your users by frequently training your LUIS model.'),
+        learnMore: '',
+        checked: false,
+        onClick: () => {
+          openLink(linkToLUEditor);
+        },
+      },
+    ];
+
+    if (hasPublishingProfile) {
+      optSteps.push({
+        key: 'connections',
+        label: formatMessage('Add connections'),
+        description: formatMessage('Connect your bot to Teams, external channels, or enable speech. Learn more'),
+        learnMore: '',
+        checked: false,
+        onClick: () => {
+          openLink(linkToConnections);
+        },
+      });
+    }
+
+    setOptionalSteps(optSteps);
+  }, [botProject]);
+
+  return (
+    <ScrollablePane styles={{ root: { marginTop: 60 } }}>
+      <div css={{ paddingTop: 20, paddingLeft: 27, paddingRight: 20 }}>
+        <ManageLuis
+          hidden={!displayManageLuis}
+          setDisplayManageLuis={setDisplayManageLuis}
+          onDismiss={hideManageLuis}
+          onGetKey={updateLuisSettings}
+          onNext={() => {
+            hideManageLuis();
+            doNextStep('luis');
+          }}
+        />
+        <ManageQNA
+          hidden={!displayManageQNA}
+          setDisplayManageQna={setDisplayManageQNA}
+          onDismiss={hideManageQNA}
+          onGetKey={updateQNASettings}
+          onNext={() => {
+            hideManageQNA();
+            doNextStep('qna');
+          }}
+        />
+
+        {highlightLUIS && (
+          <TeachingBubble
+            hasCondensedHeadline
+            headline={formatMessage('Your new bot is almost ready!')}
+            target="#luis"
+            onDismiss={() => {
+              setHighlightLUIS(false);
+            }}
+          >
+            {formatMessage('Continue setting up your development environment by adding LUIS keys.')}
+          </TeachingBubble>
+        )}
+
+        {highlightQNA && (
+          <TeachingBubble
+            hasCondensedHeadline
+            headline={formatMessage('Almost there!')}
+            target="#qna"
+            onDismiss={() => {
+              setHighlightQNA(false);
+            }}
+          >
+            {formatMessage("Just add a QnA key and you'll be ready to talk to your bot.")}
+          </TeachingBubble>
+        )}
+
+        {requiredNextSteps.length ? (
+          <div>
+            <h3 style={h3Style}>{formatMessage('Required')}</h3>
+            {requiredNextSteps.map((step) => (
+              <GetStartedTask key={step.key} step={step} />
+            ))}
+          </div>
+        ) : (
+          ''
+        )}
+
+        {recommendedNextSteps.length ? (
+          <div>
+            <h3 style={h3Style}>{formatMessage('Recommended')}</h3>
+            {recommendedNextSteps.map((step) => (
+              <GetStartedTask key={step.key} step={step} />
+            ))}
+          </div>
+        ) : (
+          ''
```

Comment:
null

Reply:
ah i was wondering the best way to do that.